graphistry/pygraphistry.py

```diff
@@ -382,6 +377,11 @@ def refresh(token=None, fail_silent=False):
                 PyGraphistry._is_authenticated = True
             return PyGraphistry.api_token()
         except Exception as e:
+            if PyGraphistry.store_token_creds_in_memory():
```

Comment:
This will not actually refresh?

Reply:
Store token creds makes a login() method that can be called, while api-token() returns the stale eve if stale